django/test/testcases.py

```diff
@@ -179,10 +179,15 @@ def setUpClass(cls):
         if cls._overridden_settings:
             cls._cls_overridden_context = override_settings(**cls._overridden_settings)
             cls._cls_overridden_context.enable()
+            cls.addClassCleanup(cls._cls_overridden_context.disable)
+            cls.addClassCleanup(delattr, cls, '_cls_overridden_context')
```

Comment:
I don't think there's a need to delete attributes now, since they're not used in `hasattr` checks. I guess deleting them helps reduce memory consumption a tiny bit though.

Reply:
True it's unnecessary. I will remove these lines.